content/strategy-goals/strategy/code-graph/code-intelligence/index.md

```diff
@@ -45,20 +45,20 @@ At the moment maturity varies depending on the area of ownership:
 In the last few months we’ve:
```

Comment:
I can't add a comment above, but I would change the sentence `Code intelligence lies at the very center of the product as a whole, providing the metadata that will eventually power other product areas.` to remove "will eventually power" and say "powers". This reality is already here.

Reply:
I really like the maturity descriptions generally.